docs/sentinel1.md

```diff
@@ -16,7 +16,7 @@ While both satellites were actively imaging, most global landmasses were imaged
 
     **As of December 23, 2021, Sentinel-1B is no longer able to acquire data.** An anomaly related to the power supply cannot be repaired, and the satellite will be decommissioned. Refer to [ESA's documentation of the end of the Sentinel-1B mission](https://www.esa.int/Applications/Observing_the_Earth/Copernicus/Sentinel-1/Mission_ends_for_Copernicus_Sentinel-1B_satellite "Mission Ends for Copernicus Sentinel-1B Satellite" ){target=_blank} for more information.
 
-While Sentinel-1A is still healthy, the loss of Sentinel-1B has resulted in a significant reduction in the spatial and temporal coverage of the Sentinel-1 mission. Refer to [this article by Iain Woodhouse](https://www.earthblox.io/articles/the-impact-of-the-sentinel-1b-failure-and-looking-forward-to-sentinel-1c "The impact of the Sentinel-1B failure and looking forward to Sentinel-1C" ){target=_blank} for an illustration of the global impact of the Sentinel-1B failure. The image below illustrates a gap in the acquisitions over Alaska. This area of the Yukon-Kuskokwim Delta did not have a Sentinel-1 acquisition during the summer of 2022 until August 15.
+While Sentinel-1A is still healthy, the loss of Sentinel-1B has resulted in a significant reduction in the spatial and temporal coverage of the Sentinel-1 mission. Refer to [this article by Ian Woodhouse](https://www.earthblox.io/articles/the-impact-of-the-sentinel-1b-failure-and-looking-forward-to-sentinel-1c "The impact of the Sentinel-1B failure and looking forward to Sentinel-1C" ){target=_blank} for an illustration of the global impact of the Sentinel-1B failure. The image below illustrates a gap in the acquisitions over Alaska. This area of the Yukon-Kuskokwim Delta did not have a Sentinel-1 acquisition during the summer of 2022 until August 15.
```

Comment:
```suggestion
While Sentinel-1A is still healthy, the loss of Sentinel-1B has resulted in a significant reduction in the spatial and temporal coverage of the Sentinel-1 mission. Refer to [this article by Iain Woodhouse](https://www.earthblox.io/articles/the-impact-of-the-sentinel-1b-failure-and-looking-forward-to-sentinel-1c "The impact of the Sentinel-1B failure and looking forward to Sentinel-1C" ){target=_blank} for an illustration of the global impact of the Sentinel-1B failure. The image below illustrates a gap in the acquisitions over Alaska. This area of the Yukon-Kuskokwim Delta did not have a Sentinel-1 acquisition during the summer of 2022 until August 15.
```

Iain's name was spelled correctly before:
![image](https://user-images.githubusercontent.com/7882693/218831100-eb9a724d-4aee-464a-b8fe-d23a3dcf7d4c.png)

Reply:
Weird. Not sure how that happened.